prql/src/ast_fold.rs

```diff
@@ -31,7 +31,7 @@ pub trait AstFold {
     fn fold_pipeline(&mut self, pipeline: Vec<Transform>) -> Result<Vec<Transform>> {
         pipeline
             .into_iter()
-            .map(|t| self.fold_transformation(t))
+            .map(|t| self.fold_transform(t))
```

Comment:
Shall we call these `transforms` everywhere? I'm up for it if you are...

Reply:
Oh, forgot to tell you about that: I've changed that in a few places. It seemed better name, just because its shorter.

We should go over the project and refacator it everywhere

Reply:
I agree! 

https://en.wikipedia.org/wiki/Nominalization is impossible to stop!